package.json

```diff
@@ -162,7 +162,7 @@
     "babel-plugin-dynamic-import-node": "^2.2.0",
     "csstype": "^3.0.8",
     "eslint": "5.11.1",
-    "eslint-config-sentry-app": "1.67.0",
+    "eslint-config-sentry-app": "1.69.0",
```

Comment:
@vuluongj20 let's do this in a separate PR - you'll want to do `yarn add eslint-config-sentry-app@1.69.0` because you'll need to update the yarn lockfile as well. We also need to do this on the getsentry repo.